targets/mbedos5/jerryscript-mbed/jerryscript-mbed-drivers/source/AnalogIn-js.cpp

```diff
@@ -28,10 +46,16 @@ DECLARE_CLASS_FUNCTION(AnalogIn, read) {
     CHECK_ARGUMENT_COUNT(AnalogIn, read, (args_count == 0));
 
     // Extract native AnalogIn pointer
-    uintptr_t ptr_val;
-    jerry_get_object_native_handle(this_obj, &ptr_val);
+    void* ptr_val;
+    const jerry_object_native_info_t *type_ptr;
```

Comment:
in these two consecutive lines, pointer variable declarations follow two different styles: `type* var` and `type *var`. please, have a look at the rest of the code, and if one of them is used consistently and only the new additions deviate, adapt the new code. (if the code is already of heavily mixed style, then leave it)